### ISSUE TITLE
[Feature Request] Is it possible to do not show a notification when use floating button?

### ISSUE THREAD

**Issue by nodejsg** [AhmetCanArslan/ShizuWall#137]
### Describe your idea

[Feature Request] Is it possible to do not show a notification when use floating button? 
(use Android accessibility features?)

**Comment by AhmetCanArslan** [comment]
You can turn this notification off and it's category from native Android settings. It's required for keeping foreground activity alive. And also, I cannot use Accessibility since Google Play restricts its usage too much that it's nearly impossible to get accepted while using this.
